products/fairphone.md

```diff
@@ -0,0 +1,38 @@
+---
+permalink: /fairphone
+title: Fairphone
+category: device
+releasePolicyLink: https://forum.fairphone.com/t/fairphone-product-release-cycle/52652
+discontinuedColumn: true
+activeSupportColumn: false
+releaseColumn: false
+releaseDateColumn: true
+releases:
+-   releaseCycle: "Fairphone 4"
+    discontinued: false
+    eol: false
+    releaseDate: 2021-09-30
+-   releaseCycle: "Fairphone 3+"
+    discontinued: false
+    eol: 2021-09-01
+    releaseDate: 2020-09-30
+-   releaseCycle: "Fairphone 3"
+    discontinued: 2021-09-01
+    eol: 2021-09-02
+    releaseDate: 2019-09-30
+-   releaseCycle: "Fairphone 2"
+    discontinued: 2021-11-01
+    eol: 2019-03-22
+    releaseDate: 2015-12-01
+-   releaseCycle: "Fairphone 1"
```

Comment:
Use a releaseLabel to allow for easier matching for clients.

```suggestion
releaseLabel: "Fairphone __RELEASE_CYCLE__"
releases:
-   releaseCycle: "4"
    discontinued: false
    eol: false
    releaseDate: 2021-09-30
-   releaseCycle: "3+"
    discontinued: false
    eol: 2021-09-01
    releaseDate: 2020-09-30
-   releaseCycle: "3"
    discontinued: 2021-09-01
    eol: 2021-09-02
    releaseDate: 2019-09-30
-   releaseCycle: "2"
    discontinued: 2021-11-01
    eol: 2019-03-22
    releaseDate: 2015-12-01
-   releaseCycle: "1"
```

Reply:
just because of curiosity which one looks better 3+ or 3plus ?  because when i see 3+ i always think that 3 4 5 .,..